### ISSUE TITLE
improve bulkselection dialog

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#279]
```
In the bulkselection there is no need for the slider when there is only 1 item. 
Also it would be nice to have an image and the name of the item that is handled.

All this will be added with this patch.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 19 Nov 2011 at 4:25

Attachments:
- [Bulkselection - disable slider for 1 item - provide image and name of item.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-279/comment-0/Bulkselection - disable slider for 1 item - provide image and name of item.patch)

**Comment by GoogleCodeExporter** [comment]
```
Good looking work
```

Original comment by `sdeva...@gmail.com` on 19 Nov 2011 at 11:11
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Looks good, thanks Samuel. Committed in r224.
```

Original comment by `oskar.wi...@gmail.com` on 12 Mar 2012 at 10:07
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
